### PR TITLE
Add info about pycrypto and pycountry variables to install doc

### PR DESCRIPTION
**Pull request by mmetak**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -169,13 +169,26 @@ Name                                 Notes
                                      audio and video streams, eg. YouTube 1080p+
 ==================================== ===========================================
 
+Using pycrypto and pycountry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (beardypig) [inline docs/install.rst]: I'm surprised this didn't fail the build. Could you make the `^^` underline the same width and the text above?

Comment (mmetak) [inline docs/install.rst]: Sorry about that @beardypig , forgot to adjust it after I made a change. Should be fixed now.

Comment (beardypig) [inline docs/install.rst]: Thankee kindly sai.

**Comment by gravyboat** [comment]
Merging now that the underscores are fixed, thanks @mmetak!

**Pull request merged**